
        <!DOCTYPE html>
        <html lang="en">
        <head>
            <meta charset="UTF-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <title>Automated Test Generation - Generated by Lance</title>
            <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
            <link href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/themes/prism-okaidia.min.css" rel="stylesheet" />
            <style>                
                body {
                    font-family: 'Roboto', sans-serif;
                    background-color: #f8f9fa;
                    color: #333;
                    margin: 0;
                    padding: 20px;
                    }
                    header {
                        margin-bottom: 20px;
                        text-align: center;
                    }
                    header h1 {
                        margin: 0;
                        font-size: 2.5em;
                        color: #007bff;
                    }
                    header p {
                        margin: 5px 0 0;
                        font-size: 1.2em;
                        color: #6c757d;
                    }
                    table {
                        width: 100%;
                        border-collapse: collapse;
                        box-shadow: 0 2px 3px rgba(0,0,0,0.1);
                        background-color: #fff;
                    }
                    th, td {
                        padding: 15px;
                        text-align: left;
                        border: 1px solid #dee2e6;
                    }
                    th {
                        background-color: #007bff;
                        color: #fff;
                        font-weight: 700;
                    }
                    tr:nth-child(even) {
                        background-color: #f2f2f2;
                    }
                    .status-pass {
                        color: green;
                        font-weight: 700;
                    }
                    .status-fail {
                        color: red;
                        font-weight: 700;
                    }
                    pre {
                        background-color: #282a36 !important;
                        color: #f8f8f2 !important;
                        padding: 10px;
                        border-radius: 5px;
                        overflow: auto;
                    }
                    code {
                        font-family: 'Courier New', Courier, monospace;
                    }
            </style>
        </head>
        <body>
            <header>
                <h1>Test Results</h1>
                <p>Generated by Lance</p>
            </header>
            
            <table>
                <tr>
                    <th>Status</th>
                    <th>Label</th>
                    <th>Reason</th>
                    <th>Exit Code</th>
                    <th>Line Coverage</th>
                    <th>Branch Coverage</th>
                    <th>Stderr</th>
                    <th>Stdout</th>
                    <th>Test</th>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_0</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test that the constructor with 3 parameters sets the chronology correctly\n', 'test_name': 'testConstructorWith3Params', 'test_code': '@Test\npublic void testConstructorWith3Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.DateTimeZone;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_0</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test that the constructor with 5 parameters sets the pivot year correctly\n', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), 2000, 2020);\n    assertEquals((Integer) 2000, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_0</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test that the saveField method sets the field value correctly\n', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_0</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test that the computeMillis method throws an exception for an out of range field value\n', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_0</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test that the restoreState method restores the state correctly\n', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-INFO">INFO</td>
                    <td>g_0</td>
                    <td>69.54264068603516</td>
                    <td>6377</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test constructor with 3 parameters', 'test_name': 'testConstructorWith3Params', 'test_code': '@Test\npublic void testConstructorWith3Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.DateTimeZone;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test constructor with 5 parameters', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), 2000, 2020);\n    assertEquals((Integer) 2000, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test saveField method', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis out of range', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test restoreState method', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test constructor with 3 parameters', 'test_name': 'testConstructorWith3Params', 'test_code': '@Test\npublic void testConstructorWith3Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.DateTimeZone;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test constructor with 5 parameters', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), 2000, 2020);\n    assertEquals((Integer) 2000, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test saveField method', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with out of range value', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test restoreState method', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test constructor with 3 parameters', 'test_name': 'testConstructorWith3Params', 'test_code': '@Test\npublic void testConstructorWith3Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.DateTimeZone;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test constructor with 5 parameters', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), 2000, 2020);\n    assertEquals((Integer) 2000, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test saveField method', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with out of range value', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_0_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test restoreState method', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-INFO">INFO</td>
                    <td>f_0</td>
                    <td>218.83787393569946</td>
                    <td>25454</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test DateTimeParserBucket constructor with 3 parameters', 'test_name': 'testConstructorWith3Params', 'test_code': '@Test\npublic void testConstructorWith3Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.DateTimeZone;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test DateTimeParserBucket constructor with 5 parameters', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), 2000, 2000);\n    assertEquals((Integer) 2000, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test saveField method', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis method with out of range value', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test restoreState method', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getChronology method', 'test_name': 'testGetChronology', 'test_code': '@Test\npublic void testGetChronology() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getLocale method', 'test_name': 'testGetLocale', 'test_code': '@Test\npublic void testGetLocale() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(Locale.getDefault(), bucket.getLocale());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getZone method', 'test_name': 'testGetZone', 'test_code': '@Test\npublic void testGetZone() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.getZone(), bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setZone method', 'test_name': 'testSetZone', 'test_code': '@Test\npublic void testSetZone() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    DateTimeZone zone = DateTimeZone.forID("America/New_York");\n    bucket.setZone(zone);\n    assertEquals(zone, bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getOffset method', 'test_name': 'testGetOffset', 'test_code': '@Test\npublic void testGetOffset() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(0, bucket.getOffset());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setOffset method', 'test_name': 'testSetOffset', 'test_code': '@Test\npublic void testSetOffset() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.setOffset(3600);\n    assertEquals(3600, bucket.getOffset());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getPivotYear method', 'test_name': 'testGetPivotYear', 'test_code': '@Test\npublic void testGetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(null, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setPivotYear method', 'test_name': 'testSetPivotYear', 'test_code': '@Test\npublic void testSetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.setPivotYear(2020);\n    assertEquals(2020, bucket.getPivotYear().intValue());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis method with resetFields parameter', 'test_name': 'testComputeMillisResetFields', 'test_code': '@Test\npublic void testComputeMillisResetFields() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 12);\n    bucket.saveField(DateTimeFieldType.dayOfMonth(), 31);\n    long millis = bucket.computeMillis(true);\n    assertEquals(2020, (millis / 31536000000L));\n    assertEquals(12, (millis % 31536000000L) / 2628000000L);\n    assertEquals(31, (millis % 2628000000L) / 86400000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-INFO">INFO</td>
                    <td>g_1</td>
                    <td>138.5238196849823</td>
                    <td>8096</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the constructor with 3 parameters', 'test_name': 'testConstructorWith3Params', 'test_code': '@Test\npublic void testConstructorWith3Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.DateTimeZone;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the constructor with 5 parameters', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), 2000, 2000);\n    assertEquals((Integer) 2000, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the saveField method', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with an out of range value', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the restoreState method', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getChronology method', 'test_name': 'testGetChronology', 'test_code': '@Test\npublic void testGetChronology() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getLocale method', 'test_name': 'testGetLocale', 'test_code': '@Test\npublic void testGetLocale() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(Locale.getDefault(), bucket.getLocale());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getZone method', 'test_name': 'testGetZone', 'test_code': '@Test\npublic void testGetZone() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.getZone(), bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setZone method', 'test_name': 'testSetZone', 'test_code': '@Test\npublic void testSetZone() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    DateTimeZone zone = DateTimeZone.forID("America/New_York");\n    bucket.setZone(zone);\n    assertEquals(zone, bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getOffset method', 'test_name': 'testGetOffset', 'test_code': '@Test\npublic void testGetOffset() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(0, bucket.getOffset());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setOffset method', 'test_name': 'testSetOffset', 'test_code': '@Test\npublic void testSetOffset() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.setOffset(3600);\n    assertEquals(3600, bucket.getOffset());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getPivotYear method', 'test_name': 'testGetPivotYear', 'test_code': '@Test\npublic void testGetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(null, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setPivotYear method', 'test_name': 'testSetPivotYear', 'test_code': '@Test\npublic void testSetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.setPivotYear(2020);\n    assertEquals(2020, bucket.getPivotYear().intValue());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,68] error: incompatible types: DateTimeZone cannot be converted to Chronology
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with resetFields', 'test_name': 'testComputeMillisResetFields', 'test_code': '@Test\npublic void testComputeMillisResetFields() {\n    Chronology chrono = DateTimeUtils.getChronology(DateTimeZone.UTC);\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 12);\n    bucket.saveField(DateTimeFieldType.dayOfMonth(), 31);\n    long millis = bucket.computeMillis(true);\n    assertEquals(2020, (millis / 31536000000L));\n    assertEquals(12, (millis % 31536000000L) / 2628000000L);\n    assertEquals(31, (millis % 2628000000L) / 86400000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the constructor with 3 parameters', 'test_name': 'testConstructorWith3Params', 'test_code': '@Test\npublic void testConstructorWith3Params() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono, bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the constructor with 5 parameters', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), 2000, 2000);\n    assertEquals((Integer) 2000, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the saveField method', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with out of range value', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test restoreState method', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getChronology method', 'test_name': 'testGetChronology', 'test_code': '@Test\npublic void testGetChronology() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono, bucket.getChronology());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getLocale method', 'test_name': 'testGetLocale', 'test_code': '@Test\npublic void testGetLocale() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(Locale.getDefault(), bucket.getLocale());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getZone method', 'test_name': 'testGetZone', 'test_code': '@Test\npublic void testGetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(chrono.getZone(), bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setZone method', 'test_name': 'testSetZone', 'test_code': '@Test\npublic void testSetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    DateTimeZone zone = DateTimeZone.forID("America/New_York");\n    bucket.setZone(zone);\n    assertEquals(zone, bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getOffset method', 'test_name': 'testGetOffset', 'test_code': '@Test\npublic void testGetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    assertEquals(0, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setOffset method', 'test_name': 'testSetOffset', 'test_code': '@Test\npublic void testSetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.setOffset(3600);\n    assertEquals(3600, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getPivotYear method', 'test_name': 'testGetPivotYear', 'test_code': '@Test\npublic void testGetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(null, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setPivotYear method', 'test_name': 'testSetPivotYear', 'test_code': '@Test\npublic void testSetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setPivotYear(2020);\n    assertEquals(2020, (int) bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with reset fields', 'test_name': 'testComputeMillisResetFields', 'test_code': '@Test\npublic void testComputeMillisResetFields() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault());\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 12);\n    bucket.saveField(DateTimeFieldType.dayOfMonth(), 31);\n    long millis = bucket.computeMillis(true);\n    assertEquals(2020, (millis / 31536000000L));\n    assertEquals(12, ((millis % 31536000000L) / 2628000000L));\n    assertEquals(31, ((millis % 2628000000L) / 86400000L));\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test constructor with 3 parameters', 'test_name': 'testConstructorWith3Params', 'test_code': '@Test\npublic void testConstructorWith3Params() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test constructor with 5 parameters', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), 2000, 2000);\n    assertEquals((Integer) 2000, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test saveField', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with out of range value', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test restoreState', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getChronology', 'test_name': 'testGetChronology', 'test_code': '@Test\npublic void testGetChronology() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getLocale', 'test_name': 'testGetLocale', 'test_code': '@Test\npublic void testGetLocale() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(Locale.getDefault(), bucket.getLocale());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getZone', 'test_name': 'testGetZone', 'test_code': '@Test\npublic void testGetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.getZone(), bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setZone', 'test_name': 'testSetZone', 'test_code': '@Test\npublic void testSetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    DateTimeZone zone = DateTimeZone.forID("America/New_York");\n    bucket.setZone(zone);\n    assertEquals(zone, bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getOffset', 'test_name': 'testGetOffset', 'test_code': '@Test\npublic void testGetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(0, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setOffset', 'test_name': 'testSetOffset', 'test_code': '@Test\npublic void testSetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setOffset(3600);\n    assertEquals(3600, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getPivotYear', 'test_name': 'testGetPivotYear', 'test_code': '@Test\npublic void testGetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(null, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setPivotYear', 'test_name': 'testSetPivotYear', 'test_code': '@Test\npublic void testSetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setPivotYear(2020);\n    assertEquals(2020, (int) bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_1_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with resetFields', 'test_name': 'testComputeMillisResetFields', 'test_code': '@Test\npublic void testComputeMillisResetFields() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 12);\n    bucket.saveField(DateTimeFieldType.dayOfMonth(), 31);\n    long millis = bucket.computeMillis(true);\n    assertEquals(2020, (millis / 31536000000L));\n    assertEquals(12 - 1, ((millis % 31536000000L) / 2628000000L));\n    assertEquals(31 - 1, ((millis % 2628000000L) / 86400000L));\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-INFO">INFO</td>
                    <td>f_1</td>
                    <td>521.7291054725647</td>
                    <td>34078</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the constructor with 5 parameters.', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the saveField method with a DateTimeFieldType and an integer value.', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the computeMillis method with an out of range value.', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the restoreState method.', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getChronology method.', 'test_name': 'testGetChronology', 'test_code': '@Test\npublic void testGetChronology() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getLocale method.', 'test_name': 'testGetLocale', 'test_code': '@Test\npublic void testGetLocale() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(Locale.getDefault(), bucket.getLocale());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getZone method.', 'test_name': 'testGetZone', 'test_code': '@Test\npublic void testGetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.getZone(), bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setZone method.', 'test_name': 'testSetZone', 'test_code': '@Test\npublic void testSetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    DateTimeZone zone = DateTimeZone.forID("America/New_York");\n    bucket.setZone(zone);\n    assertEquals(zone, bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getOffset method.', 'test_name': 'testGetOffset', 'test_code': '@Test\npublic void testGetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(0, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setOffset method.', 'test_name': 'testSetOffset', 'test_code': '@Test\npublic void testSetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setOffset(3600);\n    assertEquals(3600, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getPivotYear method.', 'test_name': 'testGetPivotYear', 'test_code': '@Test\npublic void testGetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(null, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setPivotYear method.', 'test_name': 'testSetPivotYear', 'test_code': '@Test\npublic void testSetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setPivotYear(2020);\n    assertEquals(2020, (int) bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>g_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the computeMillis method with resetFields set to true.', 'test_name': 'testComputeMillisResetFields', 'test_code': '@Test\npublic void testComputeMillisResetFields() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 12);\n    bucket.saveField(DateTimeFieldType.dayOfMonth(), 31);\n    long millis = bucket.computeMillis(true);\n    assertEquals(2020, (millis / 31536000000L));\n    assertEquals(12 - 1, ((millis % 31536000000L) / 2628000000L));\n    assertEquals(31 - 1, ((millis % 2628000000L) / 86400000L));\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-INFO">INFO</td>
                    <td>g_2</td>
                    <td>126.5863983631134</td>
                    <td>9442</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the constructor of the DateTimeParserBucket with five parameters.', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the saveField method.', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the computeMillis method for out of range values.', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the restoreState method.', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getChronology method.', 'test_name': 'testGetChronology', 'test_code': '@Test\npublic void testGetChronology() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getLocale method.', 'test_name': 'testGetLocale', 'test_code': '@Test\npublic void testGetLocale() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(Locale.getDefault(), bucket.getLocale());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getZone method.', 'test_name': 'testGetZone', 'test_code': '@Test\npublic void testGetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.getZone(), bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setZone method.', 'test_name': 'testSetZone', 'test_code': '@Test\npublic void testSetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    DateTimeZone zone = DateTimeZone.forID("America/New_York");\n    bucket.setZone(zone);\n    assertEquals(zone, bucket.getZone());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getOffset method.', 'test_name': 'testGetOffset', 'test_code': '@Test\npublic void testGetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(0, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setOffset method.', 'test_name': 'testSetOffset', 'test_code': '@Test\npublic void testSetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setOffset(3600);\n    assertEquals(3600, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getPivotYear method.', 'test_name': 'testGetPivotYear', 'test_code': '@Test\npublic void testGetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(null, bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setPivotYear method.', 'test_name': 'testSetPivotYear', 'test_code': '@Test\npublic void testSetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setPivotYear(2020);\n    assertEquals(2020, (int) bucket.getPivotYear());\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_1</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[39,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the computeMillis method with resetFields.', 'test_name': 'testComputeMillisResetFields', 'test_code': '@Test\npublic void testComputeMillisResetFields() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 12);\n    bucket.saveField(DateTimeFieldType.dayOfMonth(), 31);\n    long millis = bucket.computeMillis(true);\n    assertEquals(2020, (millis / 31536000000L));\n    assertEquals(12 - 1, ((millis % 31536000000L) / 2628000000L));\n    assertEquals(31 - 1, ((millis % 2628000000L) / 86400000L));\n}\n', 'new_imports_code': None}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the constructor with 5 parameters', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the saveField method', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the computeMillis method with out of range values', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the restoreState method', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getChronology method', 'test_name': 'testGetChronology', 'test_code': '@Test\npublic void testGetChronology() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getLocale method', 'test_name': 'testGetLocale', 'test_code': '@Test\npublic void testGetLocale() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(Locale.getDefault(), bucket.getLocale());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getZone method', 'test_name': 'testGetZone', 'test_code': '@Test\npublic void testGetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.getZone(), bucket.getZone());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setZone method', 'test_name': 'testSetZone', 'test_code': '@Test\npublic void testSetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    DateTimeZone zone = DateTimeZone.forID("America/New_York");\n    bucket.setZone(zone);\n    assertEquals(zone, bucket.getZone());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getOffset method', 'test_name': 'testGetOffset', 'test_code': '@Test\npublic void testGetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(0, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setOffset method', 'test_name': 'testSetOffset', 'test_code': '@Test\npublic void testSetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setOffset(3600);\n    assertEquals(3600, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the getPivotYear method', 'test_name': 'testGetPivotYear', 'test_code': '@Test\npublic void testGetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(null, bucket.getPivotYear());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the setPivotYear method', 'test_name': 'testSetPivotYear', 'test_code': '@Test\npublic void testSetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setPivotYear(2020);\n    assertEquals(2020, (int) bucket.getPivotYear());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_2</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test the computeMillis method with resetFields', 'test_name': 'testComputeMillisResetFields', 'test_code': '@Test\npublic void testComputeMillisResetFields() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 12);\n    bucket.saveField(DateTimeFieldType.dayOfMonth(), 31);\n    long millis = bucket.computeMillis(true);\n    assertEquals(2020, (millis / 31536000000L));\n    assertEquals(12 - 1, ((millis % 31536000000L) / 2628000000L));\n    assertEquals(31 - 1, ((millis % 2628000000L) / 86400000L));\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test constructor with 5 parameters', 'test_name': 'testConstructorWith5Params', 'test_code': '@Test\npublic void testConstructorWith5Params() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test saveField method', 'test_name': 'testSaveField', 'test_code': '@Test\npublic void testSaveField() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with out of range value', 'test_name': 'testComputeMillisOutOfRange', 'test_code': '@Test(expected = IllegalFieldValueException.class)\npublic void testComputeMillisOutOfRange() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 13);\n    bucket.computeMillis();\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test restoreState method', 'test_name': 'testRestoreState', 'test_code': '@Test\npublic void testRestoreState() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    Object state = bucket.saveState();\n    bucket.restoreState(state);\n    assertEquals(2020, bucket.computeMillis() / 31536000000L);\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getChronology method', 'test_name': 'testGetChronology', 'test_code': '@Test\npublic void testGetChronology() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.withUTC(), bucket.getChronology());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getLocale method', 'test_name': 'testGetLocale', 'test_code': '@Test\npublic void testGetLocale() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(Locale.getDefault(), bucket.getLocale());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getZone method', 'test_name': 'testGetZone', 'test_code': '@Test\npublic void testGetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(chrono.getZone(), bucket.getZone());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setZone method', 'test_name': 'testSetZone', 'test_code': '@Test\npublic void testSetZone() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    DateTimeZone zone = DateTimeZone.forID("America/New_York");\n    bucket.setZone(zone);\n    assertEquals(zone, bucket.getZone());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getOffset method', 'test_name': 'testGetOffset', 'test_code': '@Test\npublic void testGetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(0, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setOffset method', 'test_name': 'testSetOffset', 'test_code': '@Test\npublic void testSetOffset() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setOffset(3600);\n    assertEquals(3600, (int) bucket.getOffsetInteger());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test getPivotYear method', 'test_name': 'testGetPivotYear', 'test_code': '@Test\npublic void testGetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    assertEquals(null, bucket.getPivotYear());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test setPivotYear method', 'test_name': 'testSetPivotYear', 'test_code': '@Test\npublic void testSetPivotYear() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.setPivotYear(2020);\n    assertEquals(2020, (int) bucket.getPivotYear());\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-FAIL">FAIL</td>
                    <td>f_2_3</td>
                    <td>Compilation failure</td>
                    <td>1</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td><pre><code class="language-shell">[ERROR] /home/sijia/test-generation-coverage/guided-test-generation/defects4j-subjects-notests/Time-13f/src/test/java/org/joda/time/format/DateTimeParserBucketTest.java:[41,41] error: cannot find symbol
</code></pre></td>
                    <td><pre><code class="language-python">{'test_behavior': 'Test computeMillis with resetFields', 'test_name': 'testComputeMillisResetFields', 'test_code': '@Test\npublic void testComputeMillisResetFields() {\n    Chronology chrono = DateTimeUtils.getChronologyISO();\n    DateTimeParserBucket bucket = new DateTimeParserBucket(0, chrono, Locale.getDefault(), null, 2000);\n    bucket.saveField(DateTimeFieldType.year(), 2020);\n    bucket.saveField(DateTimeFieldType.monthOfYear(), 12);\n    bucket.saveField(DateTimeFieldType.dayOfMonth(), 31);\n    long millis = bucket.computeMillis(true);\n    assertEquals(2020, (millis / 31536000000L));\n    assertEquals(12 - 1, ((millis % 31536000000L) / 2628000000L));\n    assertEquals(31 - 1, ((millis % 2628000000L) / 86400000L));\n}\n', 'new_imports_code': 'import org.joda.time.Chronology;\nimport org.joda.time.DateTimeUtils;\n'}</code></pre></td>
                </tr>
                
                <tr>
                    <td class="status-INFO">INFO</td>
                    <td>f_2</td>
                    <td>501.0682122707367</td>
                    <td>35444</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                
                <tr>
                    <td class="status-INFO">INFO</td>
                    <td></td>
                    <td></td>
                    <td>0</td>
                    <td>0.0</td>
                    <td>0.0</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                
            </table>
            <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/prism.min.js"></script>
        </body>
        </html>
        